app/javascript/packs/document-capture.tsx

```diff
@@ -34,6 +34,8 @@ interface AppRootData {
   exitUrl: string;
   idvInPersonUrl?: string;
   securityAndPrivacyHowItWorksUrl: string;
+  skipDocAuth: string;
```

Comment:
Confirming that you still want it to be a string here?

Reply:
I thought that this was for the string that was coming from ruby but testing it, it looks like I can delete it.  The few specs (and manually testing) - it seems okay to delete but will test when I can run all specs.

Reply:
I'd expect this to be here if we're passing a `data-` attribute from the server-rendered template.